WalletWasabi.Documentation/Guides/InstallInstructions.md

```diff
@@ -110,58 +113,58 @@ sudo gpg2 --verify {file name}.asc {file name}.dmg
 
 Example: `gpg2 --verify Wasabi-1.1.6.dmg.asc Wasabi-1.1.6.dmg`.
 
-If the message returned says `Good signature from nopara73 aka Ficsór Ádám` and that it was signed with `Primary key fingerprint: 21D7 CA45 565D BCCE BE45  115D B4B7 2266 C47E 075E`, then the software was not tampered with since the developer signed it.
+If the message returned says `Good signature from nopara73 aka Ficsór Ádám` and that it was signed with Primary key fingerprint: `21D7 CA45 565D BCCE BE45  115D B4B7 2266 C47E 075E`, then the software was not tampered with since the developer signed it.
  
 Remember to check again the PGP signature every time you make a new download.
 
 If you trust nopara73's key and you are familiar with the [Web Of Trust](https://security.stackexchange.com/questions/147447/gpg-why-is-my-trusted-key-not-certified-with-a-trusted-signature), please consider also [validating it](https://www.gnupg.org/gph/en/manual/x334.html).
 
-## GPG Verification with GUI
+## GPG Verification with GUI (Windows)
 
 If you prefer the Graphical User Interface, this guide is yours. There is also a nice video guide [here](https://youtu.be/D8U53PFEsVk?t=45). 
 
-1. Download Gpg4win from https://www.gnupg.org/download/index.html
-2. Install Gpg4win 
+1. Download Gpg4win from https://www.gnupg.org/download/index.html.
+2. Install Gpg4win.
 
 ![Install Gpg4win](https://i.imgur.com/YKDdw1k.png)
 
-3. Download Wasabi latest __release__ and the corresponding __.asc__ file.
+3. Download Wasabi's latest __release__ and the corresponding __.asc__ file.
 
-4. Double click on .asc file or right click More GpgEX options / Verify. If the context menu is missing - restart the computer.
+4. Double click on .asc file or right click it and navigate to More GpgEX options and click Verify. If the context menu is missing restart the computer.
 
 ![](https://i.imgur.com/fJME8Yh.png)
 
 5. Press Search.
 
 ![](https://i.imgur.com/cj00rev.png)
 
-6. Wait until Ficsór Ádám's certificate (adam.ficsor73@gmail.com) appears. Who is this guy? The owner of WasabiWallet AKA [nopara73]( https://github.com/nopara73)
+6. Wait until Ficsór Ádám's certificate (adam.ficsor73@gmail.com) appears. Who is this guy? The owner of WasabiWallet AKA [nopara73](https://github.com/nopara73).
 
 ![](https://i.imgur.com/B3WZn1n.png)
 
-7. Add Ádám's certificate. (On the next release you can skip previous steps because the certificate will be there).
+7. Add Ádám's certificate. (Next time you want to verify a new release you can skip the previous steps because the certificate will be there).
```

Comment:
As it stands now, the brackets are redundant, imo. Either remove them, or make the whole a part of the previous sentence: 

> Add Ádám's certificate (next time you want to verify a new release you can skip the previous steps because the certificate will be there).